### PR TITLE
{lang}[intel/2017a] R v3.4.0 with extension updates to match upstream sources

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@verdurin Please update the PR title to better reflect the suggested changes.

We usually refrain from updating extension versions for R libraries unless there's a compelling reason for it (e.g. #4640)...

**Comment by verdurin** [comment]
@boegel - yes, these are the changes I needed to make for the build to succeed.

I also had a problem with `Rmpi`, which I'll mention separately.

**Comment by boegel** [comment]
@verdurin I understand the `--with-x=yes`, but why are the version bumps for the handful of extensions required exactly?

**Comment by verdurin** [comment]
@boegel because the original ones weren't downloadable anymore...

File: easybuild/easyconfigs/r/R/R-3.4.0-intel-2017a-X11-20170314.eb [hunk 1]
```diff
@@ -104,7 +104,7 @@ exts_list = [
     ('markdown', '0.8', ext_options),
     ('mlbench', '2.1-1', ext_options),
     ('NLP', '0.1-10', ext_options),
-    ('mclust', '5.2.3', ext_options),
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.4.0-intel-2017a-X11-20170314.eb]: @verdurin still available at https://cran.r-project.org/src/contrib/Archive/mclust/, which should be considered by EB (see definition of `ext_options`)?

**Comment by boegel** [comment]
@verdurin `https://cran.r-project.org/src/contrib/Archive/%(name)s` is also considered as a source URL, older package versions should still be available there?

**Comment by verdurin** [comment]
@boegel indeed, but that wasn't working for me. Could be site-specific download problems (wouldn't be the first time), but thought I should submit in case they affected other people.

**Comment by boegel** [comment]
It's likely it's a problem with some local mirror...

In any case, bumping the versions is not a permanent solution, as soon as one or more extensions get a version bump, you'll run into the same issue.

**Comment by verdurin** [comment]
Let's forget the extension updates, then.

What do you think about changing the `x` default? Certainly a lot of our users want to look at plots interactively, as well as offline.

**Comment by boegel** [comment]
@hajgato: Did you have a reason to use `--with-x=no` even though X11 is included as a dependency to R? If not, I'm all for enabling `--with-x=yes`.

**Comment by hajgato** [comment]
@boegel ???

**Comment by boegel** [comment]
@hajgato I think you were the first person to issue a PR for R with X11 included as a dep? See https://github.com/hpcugent/easybuild-easyconfigs/pull/4359/files#diff-8d7dc43a43f6155bd70784ee9c39d9e0R15

**Comment by hajgato** [comment]
@boegel: than thats an absolute mistake, it must be `-with-x=yes`

**Comment by boegel** [comment]
Thanks for the feedback @hajgato!

@verdurin Let's open a separate PR that just changes the `--with-x=yes` in all R easyconfigs that include X11 as a dep?

**Comment by verdurin** [comment]
@boegel okay, will do.

**Comment by boegel** [comment]
@vanzod ^

**Comment by boegel** [comment]
fixed in #4701

**Pull request closed without merging**